CMakeLists.txt

```diff
@@ -23,6 +23,12 @@ endif()
 
 #===========================================================================================
 # Strip garbage
+if (WIN32)
+    set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -DWIN32_CONSOLE")
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -DWIN32_CONSOLE")
```

Comment:
You don't need these once you already set the CMAKE_C_FLAGS / CMAKE_CXX_FLAGS, content of these variables will be included in all builds.